README.md

```diff
@@ -22,7 +22,33 @@ Alternatively you can install with Golang > 1.18 with:
 go install github.com/openaq/openaq-cli
 ```
 
+Homebrew Installation: 
```

Comment:
Let add a bit more narrative to each of these installation methods i.e. Who should use Homebrew? why use homebrew? which platforms?  Put yourself in the shoes of someone who isnt familiar with these and want to understand the best way to get the tool installed